frontend/src/features/crawl-workflows/workflow-editor.ts

```diff
@@ -88,6 +88,7 @@ type NewCrawlConfigParams = WorkflowParams & {
   runNow: boolean;
   config: WorkflowParams["config"] & {
     seeds: Seed[];
+    selectLinks: string[] | string | null;
```

Comment:
Any reason for it to not always be an array of strings or empty array?

Reply:
It's this way on the backend (similar to include/exclude on the config) so that the field is automatically converted into the right crawler CLI args. I think in the frontend we might always want to pass it as an array with `a[href]->href` as the first value, and just allow users to add additional selectors if they want?

Reply:
It's a good push though - I'll take a look at whether we can simplify the types for this tomorrow